data.json

```
@@ -179,6 +179,15 @@
     "dateAdded": "2018-10-23T10:17:39.000Z",
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData",
+    "difficulty": "medium",
+    "description": "At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
+    "reference": "https://blog.mayadata.io/mayadata-celebrates-hacktoberfest-2019",
+    "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest-11-1.png",
+    "dateAdded": "2018-03-11T16:47:14.000Z",
```

```suggestion
    "dateAdded": "2018-09-30T16:47:00.000Z",
```

This date should be a lot closer to today

---

👍